readme.md

```diff
@@ -305,11 +305,11 @@
 
 ## Editors
 
-- [Sublime Text](https://github.com/dreikanter/sublime-bookmarks)
-- [Vim](https://github.com/mhinz/vim-galore)
-- [Emacs](https://github.com/emacs-tw/awesome-emacs)
+- [Sublime Text](https://github.com/dreikanter/sublime-bookmarks)- Sophisticated text editor for code, markup and prose.
+- [Vim](https://github.com/mhinz/vim-galore) - Text editor that seeks to provide the power of the de-facto Unix editor 'Vi'.
```

Comment:
While vim was originally based on vi, it has evolved considerably since and so this description is not accurate.